client/me/privacy/main.jsx

```diff
@@ -56,23 +56,66 @@ const Privacy = createReactClass( {
 			TRACKS_OPT_OUT_USER_SETTINGS_KEY
 		);
 
+		const cookiePolicyLink = (
+			<a href="https://www.automattic.com/cookies" target="_blank" rel="noopener noreferrer" />
+		);
+		const privacyPolicyLink = (
+			<a href="https://www.automattic.com/privacy" target="_blank" rel="noopener noreferrer" />
+		);
+
 		return (
 			<Main className="privacy">
 				<PageViewTracker path="/me/privacy" title="Me > Privacy" />
-				<DocumentHead title={ this.props.translate( 'Privacy Settings' ) } />
+				<DocumentHead title={ translate( 'Privacy Settings' ) } />
 				<MeSidebarNavigation />
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
+				<SectionHeader label={ translate( 'Usage Information' ) } />
 				<Card className="privacy__settings">
 					<form onChange={ markChanged } onSubmit={ this.submitForm }>
+						<p>{ translate( 'We are committed to your privacy and security.' ) }</p>
+
 						<FormFieldset>
-							<FormLegend>{ translate( 'Usage Statistics' ) }</FormLegend>
-							<FormToggle
-								id="tracks_opt_out"
-								checked={ isSendingTracksEvent }
-								onChange={ this.updateTracksOptOut }
-							>
-								{ translate( 'Send usage statistics to help us improve our products.' ) }
-							</FormToggle>
+							<p>
+								<FormToggle
+									id="tracks_opt_out"
+									checked={ isSendingTracksEvent }
+									onChange={ this.updateTracksOptOut }
+								>
+									{ translate(
+										// eslint-disable-next-line max-len
```

Comment:
I'd avoid the various `eslint-disable-next-line max-len` and instead split those long strings into multiple substrings concatenated with the `+` operator, as suggested by the [i18n-calypso docs](https://github.com/Automattic/i18n-calypso#strings-only).

Reply:
Don't forget to remove this comment too!